_delb/xpath.py

```diff
@@ -13,9 +13,11 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+""" This is not an XPath implementation. """
```

Comment:
ah ok I thought this was a joke at first.

Reply:
let's see where this ends up. it wouldn't be too stupid to aim more correctness b/c at some point an XPath parser and evaluator needs to be implemented in Rust. it's stupid for many other reasons though.